Automattic/wp-calypso
# Try/full jetpack cloud

#### Changes proposed in this Pull Request

This PR is an experiment of what the full jetpack cloud with a few calypso sections(pages) might be like. 
<img width="1394" alt="Screen Shot 2020-02-12 at 4 59 19 PM" src="https://user-images.githubusercontent.com/115071/74352583-0536fe00-4db9-11ea-949f-a5724843b49e.png">


#### Testing instructions
1. add `127.0.0.1 jetpack.cloud.localhost` to your /etc/hosts file if you don't have this already. 

2. Start calypso with `npm start`
After a few minutes you should be able to access 
`jetpack.cloud.localhost:3000` in your browser.

Navigate around.
